data/questionnaires/brain-development-and-how-children-learn.yml

```diff
@@ -321,9 +309,9 @@ brain-development-and-how-children-learn:
         label: |
           Complete the statement: BLANK is the term used to describe when a more knowledgeable other helps children to acquire new skills and knowledge.
         assessment_summary: |
-          Scaffolding is where support and techniques are used to guide a child’s learning. This help can then be reduced until the child is able to complete the task alone. 
+          Scaffolding is where support and techniques are used to guide a child’s learning, Scaffolding could include modelling, chunking knowledge to make it achievable and using visual aids and questions to support learning. This help can then be reduced until the child is able to complete the task alone. 
```

Comment:
after 'guide a child's learning' there is currently a comma. I think this should be a full stop